File: 297.py
x = float(input())
n = int(input())
def expo(x, n):
    num = 1
    den = 1
    sumi = 1
    if n == 1:
        return 1
    for cnt in range(1, n):
        num *= x
        den *= cnt
        sumi += num / den
    return sumi
print("{:.3f}".format(expo(x, n) ))

File: 297.cpp
//
//  297.cpp
//  
//
//  Created by SS D on 2017-08-22.
//
//

#include <iostream>

using namespace std;

int main(){
    double sum = 1;
    int n;
    double x, num = 1, den = 1;
    cin >> x >> n;
    
    for (int cnt = 1; cnt < n; ++cnt) {
        num *= x;
        den *= cnt;
        sum += num / den;
    }
    printf("%.3lf", sum);
    return 0;
}
